content/tracing/setup/ruby.md

```diff
@@ -18,69 +21,215 @@ further_reading:
 
 ## Installation
 
-To begin tracing applications written in Ruby, first [install and configure the Datadog Agent](/tracing/setup) (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+The following steps will help you quickly start tracing your Ruby application.
+
+#### Setup the Datadog Agent
```

Comment:
nit - why do we jump from H2 to H4?

Reply:
Because the visual difference between H4 and H3 is almost indiscernible.